packages/api-generator/src/index.js

```diff
@@ -262,3 +263,29 @@ components['$vuetify'] = map['$vuetify']
 components['internationalization'] = map['internationalization']
 
 writeApiFile({ ...components, ...directives }, 'dist/api.js')
+
+// Create web-types.json to provide autocomplete in JetBrains IDEs
+const webTypes = {
+  // $schema: "../../schema/web-types.schema.json",
```

Comment:
@KaelWD should the copy paste schema file from `web-types` repo?

Reply:
https://github.com/JetBrains/web-types/issues/2

Reply:
OK, subscribed to that issue, I will get notified of activity on the issue now. :tada: